locales/cy/translations.json

```diff
@@ -64,21 +64,23 @@
     "address": {
         "chooseAddress": { 
             "addressLink": "WELSH - The address is not on the list",
-
+            "publicRegisterTitle": "WELSH - What information we'll show on the public register",
```

Comment:
we seem to have a lot of the same public register titles and text so will remove duplicates in a separate PR